src/ontology/mondo-edit.obo

```diff
@@ -262670,8 +262669,8 @@ xref: GARD:15447 {source="OMIM:609039"}
 xref: OMIM:609039 {source="MONDO:equivalentTo"}
 xref: Orphanet:2073 {source="OMIM:609039"}
 xref: UMLS:C1836907 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:609039"}
-is_a: MONDO:0016158 {source="DC-OMIM:609039"} ! narcolepsy-cataplexy syndrome
-is_a: MONDO:0021107 {source="MONDO:Redundant", source="OMIM:609039"} ! narcolepsy
+is_a: MONDO:0021107 {source="MONDO:Redundant", source="OMIM:609039"} ! hereditary narcolepsy
+relationship: excluded_subClassOf MONDO:0016158 {source="DC-OMIM:609039", source="https://orcid.org/0000-0001-5208-3432"} ! narcolepsy-cataplexy syndrome
```

Comment:
I think this term can have 2 parents: hereditary narcolepsy AND narcolepsy-cataplexy